sci-physics/lightspeed/lightspeed-1.2a-r2.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools eutils
+
+DEB_PATCH="${PN}_${PV}-7"
+DESCRIPTION="OpenGL interactive relativistic simulator"
+HOMEPAGE="http://lightspeed.sourceforge.net/"
+SRC_URI="
+	mirror://sourceforge/${PN}/${P}.tar.gz
+	mirror://sourceforge/${PN}/objects-1.tar.gz
+	mirror://debian/pool/main/${PN:0:1}/${PN}/${DEB_PATCH}.diff.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+
+IUSE="nls truetype"
+LANGS="es"
+for i in ${LANGS}; do
+	IUSE="${IUSE} linguas_${i}"
```

Comment:
Debian added Spanish localization in 2001, and no other localization has been added since then. So it seems safe to remove this loop (with its global namespace pollution) and the one below. Also it would be good to move from `linguas_*` to `l10n_*`.